daemon/daemon.go

```diff
@@ -980,6 +985,10 @@ func initNetworkController(config *Config) (libnetwork.NetworkController, error)
 	return controller, nil
 }
 
+func (daemon *Daemon) NetworkApiRouter() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
sure. But, importing net/http is purely incidental due to the API requirements of a dependent module and is not reflective of the daemon functionality in anyway. 

I dont have any strong opinion on this. If you still prefer to keep this under `docker/daemon`, i can move this function there. `net/http` will move along with it :)